charts/sourcegraph/README.md

```diff
@@ -5,7 +5,7 @@
 
 # Sourcegraph Helm Chart
 
-Visit the [Helm docs](https://docs.sourcegraph.com/admin/install/kubernetes/helm) for guidance on using this chart.
+Visit the [Helm docs](https://docs.sourcegraph.com/admin/install/kubernetes) for guidance on using this chart.
```

Comment:
Odd that `/kubernetes` is the link to the hel deployment notes while kustomize has its own subdomain `https://sourcegraph.com/docs/admin/deploy/kubernetes/kustomize`